addons/l10n_si/data/account.account.type.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="account.data_account_type_income" model="account.account.type">
+            <field name="code">uspeh.prihodki</field>
+            <field name="note"></field>
+            <field name="name">Uspeh - prihodki</field>
+            <field name="close_method">none</field>
+            <field name="report_type">income</field>
+        </record>
```

Comment:
Sorry, I overlooked that you actually override the standard account types. That does not seem wise for multicompany + multilocalization situations. In that case, I would suggest to *add* the account types that you need, but reuse the existing account types if you can.